migrations/20240330041510_/migration.sql

```diff
@@ -1,5 +1,5 @@
 -- CreateTable
-CREATE TABLE `User` (
+CREATE TABLE IF NOT EXISTS `User` (
```

Comment:
Let's undo the changes to this file. What happens here is that Keystone will catch it and warn you the following: 
```
The migration `20240330041510_` was modified after it was applied.

We need to reset the mysql database "ees-db" at localhost:3306.
? Do you want to continue? All data will be lost › (Y/n)
```


We want to be able to run `npm run dev` from this branch without breaking changes to our DB.